packages/graphql/src/translate/cypher-builder/clauses/Clause.ts

```diff
@@ -40,4 +42,10 @@ export abstract class Clause extends CypherASTNode {
     private getEnv(prefix?: string | EnvPrefix): CypherEnvironment {
         return new CypherEnvironment(prefix);
     }
+
+    /** Custom log for console.log */
+    [util.inspect.custom](depth: number, opts: InspectOptions): string {
+        const cypher = padBlock(this.build().cypher);
+        return `<Clause ${this.constructor.name}> """\n${cypher}\n"""`;
+    }
```

Comment:
Somewhat thrown off by this - can you explain how this works and where it is used?

Reply:
:magic_wand: Magic

This is similar to the `toString` method, but it is basically to customize how this class is shown in `console.log`

The idea is that, right now, for debugging, if you do:
```ts
const match = new CypherBuilder.Match(..)
console.log(match)
```

You get  a really unhelpful representation of the class, with this method, it will return the actual cypher that this CypherBuilder statement will generate:
```
console.log(match)
<Clause Match>
    MATCH (this0:Actor)-[:ACTED_IN]->(this1:Movie)
```